nameko/standalone/rpc.py

```diff
@@ -2,242 +2,154 @@
 
```

Comment:
Standalone RPC proxy completely refactored to use the new `nameko.rpc.Proxy` object and a `ReplyListener` that manages its own consumer and looks a lot like the service-based version at `nameko.rpc.ReplyListener`

Reply:
We use the shell fairly often, but that's all. Ideally we want someone who uses https://github.com/jessepollak/flask-nameko to test the pre-release.